### PR TITLE
Add Travis and linter files

### PR DESCRIPTION
**Pull request by mxpv**
*Description of changes:*
Add travis and linter configuration.
Travis will build all plugins and run bunch of linters.

~Note 1: Don't merge this please, want to make sure it works as expected with Travis.~
Note 2: There is no tests/coverage stuff, out of scope of this PR.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+language: go
+
+go:
+  - 1.11.x
+
+env:
+  - GO111MODULE=on
+
+install:
+  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $GOPATH/bin v1.12.3
```
Comment (samuelkarp) [inline .travis.yml]: It'd be good to move this in the Makefile, like `make get-deps` or something like that.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+script:
+  - make
+  - golangci-lint run
```
Comment (samuelkarp) [inline .travis.yml]: It'd be good to put this in the Makefile so you can just run `make lint`.

File: .golangci.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+linters:
+  disable-all: true
+  enable:
+    - govet
+    - errcheck
```
Comment (samuelkarp) [inline .golangci.yml]: I'm not sure about this one.  I think I'd rather have it off than have the extra `_ =` assignments and `//nolint:errcheck` comments.

File: .travis.yml (hunk 1 above)
Comment (mxpv) [reply .travis.yml]: Done

File: .travis.yml (hunk 2 above)
Comment (mxpv) [reply .travis.yml]: Done

File: .golangci.yml (hunk 3 above)
Comment (mxpv) [reply .golangci.yml]: Done

**Pull request merged**